src/metabase/public_settings.clj

```diff
@@ -232,3 +232,10 @@
    :types                 (types/types->parents :type/*)
    :entities              (types/types->parents :entity/*)
    :version               config/mb-version-info})
+
+;; This variable used to live at `metabase.metabot.instance/local-process-uuid`
+(defonce ^{:doc "This UUID is randomly-generated upon launch and used to identify this specific Metabase instance during this specifc
+                run. Restarting the server will change this UUID, and each server in a hortizontal cluster will have its own ID,
+                making this different from the `site-uuid` Setting. The local process UUID is used to differentiate different
+                horizontally clustered MB instances so we can determine which of them will handle MetaBot duties."}
+  local-process-uuid (str (UUID/randomUUID)))
```

Comment:
Sorry, I'm a knucklehead. I thought for some reason `local-process-uuid` was a Setting the same way `site-uuid` is. Since this is a plain UUID I think we should go ahead and put it in `metabase.config` instead. Sorry about that.

Reply:
Done :wrench: